raiden/api/rest.py

```diff
@@ -379,21 +381,15 @@ def __init__(
                     route, route, view_func=self._serve_webui, methods=("GET",)
                 )
 
-        # FIXME: decide what to do with this
-        # self._is_raiden_running()
-
-    def _is_raiden_running(self) -> None:
-        # We cannot accept requests before the node has synchronized with the
-        # blockchain, which is done during the call to RaidenService.start.
-        # Otherwise there is no guarantee that the node is in a valid state and
-        # that the actions are valid, e.g. deposit in a channel that has closed
-        # while the node was offline.
-        if not self.rest_api.raiden_api.raiden:
```

Comment:
Is this not needed anymore due to the changes in https://github.com/raiden-network/raiden/pull/5904?

Reply:
I don't think so, because we start the API now after all other steps have been run in `RaidenService.start()`:
https://github.com/raiden-network/raiden/blob/6266bdf3e517521d3f09b5b46d3721d3a63f073d/raiden/raiden_service.py#L373-L386